test/Microsoft.AspNetCore.SignalR.Client.FunctionalTests/Hubs.cs

```diff
@@ -29,6 +29,26 @@ public class TestHub : Hub
             await Clients.Client(Context.ConnectionId).InvokeAsync("Echo", message);
         }
 
+        public int CheckLocalPort()
```

Comment:
Use `Get` as in `GetLocalPort`. It fits with the other patterns (for headers/cookies, etc.)